src/main/java/frc/robot/commands/DriveUntilHitHub.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.drivetrains.commands.DriveArcade;
+import com.spikes2212.dashboard.RootNamespace;
+import frc.robot.subsystems.Drivetrain;
+
+import java.util.function.Supplier;
+
+/**
+ * Drives the robot forward until it hits the hub. Robot needs to be pre aligned to the hub.
+ */
+public class DriveUntilHitHub extends DriveArcade {
+
+    private static final RootNamespace rootNamespace = new RootNamespace("drive until hit hub");
+
+    /**
+     * A value between the stall current and running current of the motors
+     */
+    private static final Supplier<Double> STALL_CURRENT = rootNamespace.addConstantDouble("stall current", 15);
+    private static final Supplier<Double> MOVEMENT_SPEED = rootNamespace.addConstantDouble("movement speed", 0.5);
+
+    private final Drivetrain drivetrain;
```

Comment:
you already have a protected `tankDrivetrain` in the superclass, why create another one here?